x-pack/plugins/security_solution/public/management/pages/policy/view/host_isolation_exceptions/components/empty_unassigned.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiEmptyPrompt, EuiLink, EuiPageTemplate } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React from 'react';
+
+export const PolicyHostIsolationExceptionsEmptyUnassigned = ({
+  policyName,
+  toHostIsolationList,
+}: {
+  policyName: string;
+  toHostIsolationList: string;
+}) => {
+  return (
+    <EuiPageTemplate template="centeredContent">
+      <EuiEmptyPrompt
+        iconType="plusInCircle"
+        data-test-subj="policy-host-isolation-exceptions-empty-unassigned"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.hostIsolationExceptions.empty.unassigned.title"
+              defaultMessage="No assigned host isolation exceptions"
+            />
+          </h2>
+        }
+        body={
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.policy.hostIsolationExceptions.empty.unassigned.content"
+            defaultMessage="There are currently no host isolation exceptions assigned to {policyName}. Assign exceptions now or add and manage them on the host isolation exceptions page."
+            values={{ policyName }}
+          />
+        }
+        actions={[
```

Comment:
I think there is an action item missing here. Looking at the trusted apps one:

![Screenshot 2021-12-13 at 09 29 38](https://user-images.githubusercontent.com/15727784/145797338-f668512d-2531-4f08-88a5-9341878660f8.png)

User should be able to assign exceptions or manage the existing one in order to be able to create a new one right?

Reply:
As discussed this action will be added in a follow up PR to implement this functionality.